### ISSUE TITLE
Update the README with current Apple requirements

### ISSUE THREAD

**Issue by ThePyceIsRight** [tcurdt/iProxy#36]
Your readme says you need an apple developer license - is this still the case in 2023? I have Xcode and can push apps to my personal phone from my Mac.

also, does this method of proxying still work to bypass modern cell carrier restrictions? I use Mint which is now owned by T-Mobile.

**Comment by tcurdt** [comment]
Yes, I think the [requirement for the developer license was dropeed a couple years back](https://stackoverflow.com/questions/71449405/build-an-app-for-iphone-no-apple-developer-account). So it would probably be good to update the README.

Keep in mind that this is a very old project. You probably would have find a fork that is updated to work with the current tools - or do the update yourself. But that's just me guessing.

I cannot really help you with the carrier question. I don't know what restrictions they have put in place. But I cannot imagine how they could possibly block iProxy. In the end iProxy just acts as a proxy in your LAN.
